### PR TITLE
Add timeout to urlopen in order to avoid "freeze" of the cron job

### PR DESCRIPTION
**Pull request by seb-elico**

### CODE REVIEW

File: dead_mans_switch_client/models/dead_mans_switch_client.py [hunk 1]
```diff
@@ -11,6 +11,8 @@
 import urllib2
 from openerp import api, models
 
+SEND_TIMEOUT = 60
```
Comment (hbrunn) [inline dead_mans_switch_client/models/dead_mans_switch_client.py:15]: could you please make this a config parameter? Just say `self.env['ir.config_parameter'].get_param('dead_mans_switch_client.send_timeout`, 60)` below. Also drop a sentence about that in the readme

Comment (hbrunn) [inline dead_mans_switch_client/models/dead_mans_switch_client.py:15]: ah, I see below

File: dead_mans_switch_client/models/dead_mans_switch_client.py [hunk 2]
```diff
@@ -53,6 +55,10 @@ def alive(self):
         if not url:
             logger.error('No server configured!')
             return
+        timeout = self.env['ir.config_parameter'].get_param(
+            'dead_mans_switch_client.send_timeout')
```
Comment (hbrunn) [inline dead_mans_switch_client/models/dead_mans_switch_client.py]: just put the default as second parameter, then you can delete the following two lines

**Comment by seb-elico** [comment]
Thanks for the feedback. Please check it!

**Comment by StefanRijnhart** [comment]
:+1:

**Comment by hbrunn** [comment]
:+1:

**Comment by hbrunn** [comment]
@seb-elico please rebase, then we can merge

**Comment by seb-elico** [comment]
@hbrunn Rebased!

**Pull request merged**